EventGenerator/src/PionFilter_module.cc

```diff
@@ -17,7 +18,6 @@
 #include "Offline/TrackerGeom/inc/Tracker.hh"
```

Comment:
I think some of these includes aren't needed here and can be removed (e.g. geometry ones)

Reply:
will fix